recipes/stateless-service/06-task.yml

```diff
@@ -0,0 +1,111 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: build-project
+spec:
+  workspaces:
+  - name: maven-repo
+  inputs:
+    params:
+    - name: directory
+      default: .
+    - name: cache
+      default: empty-dir-volume
+    resources:
+    - name: src
+      type: git
+  steps:
+   - name: set-source-permissions
+     image: bash
+     command:
+     - chmod
+     - -R
+     - a+w
+     - .
+     workingDir: /workspace/src/$(inputs.params.directory)
+   - name: build-package
+     image: quay.io/quarkus/centos-quarkus-maven:20.1.0-java11
```

Comment:
Can't we use the Kogito Images builder here? :thinking:

Reply:
+1 we should be able to use our builder images and that way we will be able build it with s2i commands instead of maven commands any thoughts?